src/jit/codegenxarch.cpp

```diff
@@ -7587,8 +7588,14 @@ void CodeGen::genRemoveAlignmentAfterCall(GenTreeCall* call, unsigned bias)
         SubtractNestedAlignment(padStkAlign);
     }
 #else  // UNIX_X86_ABI
-    assert(bias == 0);
+    if (bias != 0)
+    {
+        inst_RV_IV(INS_add, REG_SPBASE, bias, EA_PTRSIZE);
```

Comment:
These changes are introduced to merge stack align/argument adjustment into one, but it would be better to use ``genAdjustSP`` as you suggested (to minimize the impact on x86/Windows).